pkg/batch/executor_test.go

```diff
@@ -11,6 +11,50 @@ import (
 	"github.com/treeverse/lakefs/pkg/logging"
 )
 
+const accessedOnce = 1
+const accessedTwice = 2
```

Comment:
Not a fan of explicitly naming "1" and "2": they appear in the names, and cannot be changed without changing the name and all of its occurrences (and checking at each occurrence whether it needs to be kept or changed).  E.g., if for some reason the second number had to change to 17, the change would be
```go
const accessedSeventeenTimes = 17
```
rather than
```go
const accessedTwice = 17
```
and we would _still_ need to go over each `accessedTwice` to determine how many times we now want it to be accessed.

The same holds for `accessedOnce`: wherever we use it, it's easier for me just to read the number 1.

Reply:
> cannot be changed without changing the name and all of its occurrences
Good point, I removed the constants because they are no longer used by multiple unit tests but I agree.